src/Services/privateApiService.js

```diff
@@ -6,16 +6,31 @@ const config = {
     }
 }
 
+const axiosInstance = axios.create({
+    baseURL: 'http://ongapi.alkemy.org/private/api',
+});
+
+axiosInstance.interceptors.request.use((config)=> {
+    config.headers.Authorization = getAuthorizationHeader();
+    config.headers.Group = 91;
+    return config
+})
+
+export const privateGet = async (url,id,params={}) => {
+    const idPlaceholder = id ? `/${id}` : '';
+    axiosInstance.get(`${url}${idPlaceholder}`, {params})
+};
```

Comment:
I think that you should return the get's result

Reply:
Perfect, ill change it.